The_Simple_Stable_Horde_Colab.ipynb

```diff
@@ -37,7 +37,7 @@
         "horde_url = \"https://stablehorde.net\" \n",
         "worker_name = \"Give your worker a cool name\" #@param {type:\"string\"}\n",
         "\n",
-        "if worker_name == \"\" or worker_name == \"Give me a cool worker name!\":\n",
+        "if worker_name == \"\" or worker_name == \"Give me a cool worker name\":\n",
```

Comment:
Still need to adjust to be the same as the default `worker_name` above:

```suggestion
        "if worker_name == \"\" or worker_name == \"Give your worker a cool name\":\n",
```

Reply:
Too late, it was already merged as is 😞 
Feel free to make another PR to fix this!